doc/src/devdocs/gc-sa.md

```diff
@@ -251,7 +251,9 @@ jl_ast_context_t *jl_ast_ctx(fl_context_t *fl) JL_GLOBALLY_ROOTED;
 
 This annotations is essentially equivalent to JL_GLOBALLY_ROOTED, except that
 is should only be used if those values are globally rooted by virtue of being
-a leaftype. The rooting of leaftypes is a bit complicated, and we can generally
+a leaftype. The rooting of leaftypes is a bit complicated. They are generally
+rooted through `cache` field of the corresponding `TypeName`, which itself is
+rooted by the containing module (so they're rooted as long as the containing module is ok) and we can generally
```

Comment:
I think there should be a "the" before "\`cache` field". Also, I suppose the third line should be wrapped to match its surroundings.

Reply:
Thanks. I'll include that in the next PR in this series.